### ISSUE TITLE
Query the activity endpoint without an itemid filter doesn't return any results

### ISSUE THREAD

**Issue by AlexAndBear** [owncloud/ocis#10153]
## Describe the bug
As per title, it should be possible to query the activities endpoint without any itemid filter.

**Comment by kobergj** [comment]
What would be the expected outcome? All events from all spaces? This can get quite a lot.

**Comment by AlexAndBear** [comment]
User facing outcome, or what is your question? If you have performance concerns: Later we will have server site pagination on top of that

**Comment by kobergj** [comment]
No. The question is what do you expect to get when you query the activity endpoint without an item- or spaceid?

**Comment by AlexAndBear** [comment]
To get activities on all locations (spaces)
